app/src/main/java/com/example/lexis/utilities/Utils.java

```diff
@@ -347,4 +361,97 @@ public static void saveWord(String targetLanguage, String targetWord, String eng
             }
         });
     }
+
+    /*
+    Return the URL for the profile picture of the provided ParseUser.
+    */
+    public static String getProfilePictureUrl(ParseUser user) {
+        ParseFile profilePic = user.getParseFile("profilePicture");
+        if (profilePic != null) {
+            return profilePic.getUrl();
+        }
+        return null;
+    }
+
+    /*
+    Create a Bitmap from the image at the specified URI.
+    */
+    public static Bitmap loadFromUri(Fragment fragment, Uri photoUri) {
+        Bitmap image = null;
+        try {
+            // check version of Android on device
+            if (Build.VERSION.SDK_INT > 27){
+                ImageDecoder.Source source = ImageDecoder.createSource(fragment.getActivity().getContentResolver(), photoUri);
+                image = ImageDecoder.decodeBitmap(source);
+            } else {
+                image = MediaStore.Images.Media.getBitmap(fragment.getActivity().getContentResolver(), photoUri);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return image;
+    }
+
+    /*
+    Convert the provided string representing a time into milliseconds.
+    */
+    public static long timerStringToMillis(String time) {
+        boolean hasHour = time.length() > MINUTE_STRING_LENGTH;
+
+        // construct date object from given time string
+        String myDate = hasHour ? "1970/01/01 " : "1970/01/01 0:";
+        myDate += time;
+
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd H:mm:ss", Locale.US);
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+        Date date;
+        try {
+            date = sdf.parse(myDate);
+            return date.getTime();
+        } catch (java.text.ParseException e) {
+            e.printStackTrace();
+        }
+        return 0;
+    }
+
+    /*
+    Convert the provided milliseconds into a displayable timer string.
+    */
+    public static String millisToTimerString(long millis) {
+        boolean hasHour = millis >= MILLIS_PER_HOUR;
+        String format = hasHour ? CHRONOMETER_FORMAT_HOUR : CHRONOMETER_FORMAT_MINUTE;
+        DateFormat formatter = new SimpleDateFormat(format, Locale.US);
+        formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
+        return formatter.format(new Date(millis));
+    }
+
+    /*
+    Return the best word search time (in milliseconds) for the provided ParseUser.
+    */
+    public static long getBestTime(ParseUser user) {
+        return user.getLong("bestTime"); // TODO: how to do null-checking here?
```

Comment:
As discussed in 1:1, you can simply check for the default value of primitive types. ^ Let me know if you still run into any issues here!